src/main/scala/xerial/sbt/Sonatype.scala

```diff
@@ -59,6 +59,7 @@ object Sonatype extends AutoPlugin with LogSupport {
 
   val sonatypeLegacy = "oss.sonatype.org"
   val sonatype01     = "s01.oss.sonatype.org"
+  val KnownOssHosts  = Seq(sonatypeLegacy, sonatype01)
```

Comment:
```suggestion
  val knownOssHosts  = Seq(sonatypeLegacy, sonatype01)
```
For consistency of variable naming.